src/index.html

```diff
@@ -8,6 +8,78 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container-of-stars">
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
```

Comment:
```suggestion
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
```

Add blank line only for multiple sibling blocks and not for single tags

example:

```
<li>
  <a>link1</a>
</li>

<li>
  <a>link2</a>
</li>
```